habitat_sim/simulator.py

```diff
@@ -145,6 +145,21 @@ def _config_pathfinder(self, config: Configuration):
                 f"Could not find navmesh {navmesh_filenname}, no collision checking will be done"
             )
 
+        agent_legacy_config = habitat_sim.AgentConfiguration()
```

Comment:
I'm confused how this works.  We don't have an `import habitat_sim` in this file, so that should lead to an error?

Reply:
Switched from using `habitat_sim.*`.